### PR TITLE
OSX Platform Release Preparation (Cordova 9)

### PR DESCRIPTION
**Pull request by erisu**
### Platforms affected
macOS - OSX

### What does this PR do?
This PR contains final preparations for the Cordova 9 release goals. 

See [Cordova 9 Release Plan](https://github.com/apache/cordova/issues/10).
 
- Bumped `cordova-common@^3.1.0`
- Bumped ESLint dependencies with correction
  - eslint@^5.12.0
  - eslint-config-semistandard@^13.0.0
  - eslint-config-standard@^12.0.0
  - eslint-plugin-node@^8.0.1
  - eslint-plugin-promise@^4.0.1
  - eslint-plugin-standard@^4.0.0
- Updated Dependencies
  - `nopt@^4.0.1`
- Added `bin/lib/**` to nyc code coverage
- Updated Travis to run only `npm t` that covers ESLint + coverage (testing)

### What testing has been done on this change?
- `npm t`
  - ESLint
  - Coverage (component & objc testing)
- [Travis CI](https://travis-ci.org/erisu/cordova-osx/builds/478189512)
- Platform Add & Prepare
  ```
  $ npx cordova@nightly create osxTest com.foobar.osxTest
  $ cd ./osxTest
  $ npx cordova@nightly platform add github:erisu/cordova-osx\#cordova9-prep
  $ npx cordova@nightly run osx
  $ npx cordova@nightly build osx
  ```

### CODE REVIEW

**Comment by erisu** [comment]
**Remaining Task**
 - [ ] Update Cordova dependency `xcode@^2.0.0` once released.

File: package.json [hunk 1]
```diff
@@ -15,8 +18,7 @@
     "cordova:platform"
   ],
   "scripts": {
-    "cover": "npm run test:coverage",
-    "test": "npm run eslint && npm run test:component && npm run test:objc",
+    "test": "npm run eslint && npm run test:coverage",
```
Comment (dpogue) [inline package.json]: Did we intentionally drop the Objective-C test cases here?

Comment (erisu) [reply package.json]: Yes, I dropped it intentionally. test:coverage runs this config file. 

https://github.com/apache/cordova-osx/blob/01b31dc9a01321ee5aeba781448cb7359b440ca2/tests/spec/coverage.json

It covers both component and objc tests.

```
"component/**/*[sS]pec.js",
"objc/**/*[sS]pec.js"
```

Comment (dpogue) [reply package.json]: Ahh, okay

**Pull request merged**